### DISCUSSION TITLE
Additions to the About section

### DISCUSSION THREAD

**Discussion by eitancaspi** [Cookie-AutoDelete/Cookie-AutoDelete#1308] [Ideas / Feature Requests]
Hi,

Please add to the "About" section:

1. A link that will open the release notes page of this software, https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/releases - so we can quickly reach this page to learn what was changed recently
2. The date of the currently installed version, next to the version's number

Thanks!

**Comment by kennethtran93** [comment]
In the CAD web extension, you'll find the last 5 release notes on the welcome page, and a link is also available at the bottom of those release notes.  Let me know if you think the release notes should go into the about section instead.

I'll try to remember to add the date to the next released version.
